packages/perseus-editor/src/widgets/grapher-editor.tsx

```diff
@@ -7,6 +7,7 @@ import {
     containerSizeClass,
     getInteractiveBoxFromSizeClass,
 } from "@khanacademy/perseus";
+import {GrapherUtil as CoreGrapherUtil} from "@khanacademy/perseus-core";
```

Comment:
Why do these "util" functions need to be in core? Previously, I think `perseus-core` didn't have any widget-specific code. Could the utils move to `perseus-score` instead? I guess I'm confused about the purposes of `perseus-core` vs. `perseus-score`.